src/alert.h

```diff
@@ -46,7 +46,7 @@ class CUnsignedAlert
     std::string strStatusBar;
     std::string strReserved;
 
-    ADD_SERIALIZE_METHODS;
+    ADD_SERIALIZE_METHODS
```

Comment:
Style nit: 

ADD_SERIALIZE_METHODS is a macro which substitutes in a self-sufficient statement or block with its own ending semicolon or {} pair, and which therefore does not need the trailing semicolon here? If so, then the semicolon that you removed has no use to the compiler, but it does help with readability, as it indicates to the reader that the ADD_SERIALIZE_METHODS macro does not syntactically interact with the template inline void SerializationOp method below. 

Other files (wallet.h, walletdb.h, crypter.h, undo.h) also have ADD_SERIALIZE_METHODS;, and use the semicolon. I think it would be better to not make modifications to that stylistic element in one place that is inconsistent with other instances of the same idiom, and that we also shouldn't change lines without good reason for the sake of mergeability of pulls from Core.